app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+  belongs_to :user, foreign_key: :users_id
+
```

Comment:
**OPTIONAL**

- [ ] It'd be nice if you used the `after_save` to call the `update_post_user_counter` method every time a post has been updated and saved to the database.
`after_save :update_post_counter`

_Please consider the same logic in the comment and like models._